rust/src/lib/nm/settings/connection.rs

```diff
@@ -418,7 +418,9 @@ pub(crate) fn gen_nm_conn_setting(
         nm_conn_set.iface_name = None;
     }
     nm_conn_set.autoconnect = Some(true);
-    nm_conn_set.autoconnect_ports = if iface.is_controller() {
+    nm_conn_set.autoconnect_ports = if iface.is_controller()
+        || iface.iface_type() == InterfaceType::Other("ovs-port".to_string())
+    {
```

Comment:
The motivation of the change is to warrant the same behavior as NetworkManager ?

Reply:
The `nmstate::InterfaceType` does not have OVS Port, so the `is_controller()` always return false for OVS port, that's why we need special care of OVS port here.

Not sure what you are referring, but this PR is depend on recently merged NM patch.